src/cijoe/cli/cli.py

```diff
@@ -83,6 +84,18 @@ def cli_integrity_check(args):
     return 0
 
 
+def cli_script(args):
+    """Create python script with the boiler-plate code necessary for running cijoe as a standalone script."""
+    filename = "cijoe-script.py"
+    if os.path.exists(filename):
+        sys.exit("Error: File 'cijoe-script.py' already exists.")
+
+    cijoe_path = str(cijoe.__path__[0])
```

Comment:
At this point you could now go all the way and do a simple file-copy, e.g. ``shutil.copyfile()``.
To locate the script itself, then you can place the script in ``cijoe/src/cijoe/core/scripts/``, and then you can rely on the regular resource-locator.